rotkehlchen/chain/ethereum/compound.py

```diff
@@ -509,12 +514,20 @@ def _get_comp_events(
         for event in comp_events:
             timestamp = self.ethereum.get_event_timestamp(event)
             amount = token_normalized_value(hexstr_to_int(event['data']), A_COMP)
-            usd_price = query_usd_price_zero_if_error(
-                asset=A_COMP,
-                time=timestamp,
-                location='comp_claim',
-                msg_aggregator=self.msg_aggregator,
-            )
+            if timestamp > COMP_STARTING_HISTOHOUR_TS:
```

Comment:
This whole logic should not be here. This should be inside cryptocompare module itself. What if we are querying COMP price from somewhere else?